x-pack/test/alerting_api_integration/spaces_only/tests/actions/type_not_enabled.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+const PREWRITTEN_ACTION_ID = 'uuid-actionId';
+const DISABLED_ACTION_TYPE = 'test.not-enabled';
+
+// eslint-disable-next-line import/no-default-export
+export default function typeNotEnabledTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  describe('actionType not enabled', () => {
```

Comment:
ya, I actually started there - `security_and_spaces` - but then realized I’d need to create multiple documents in spaces, or something, and then thought - this would be way easier to do in `spaces_only`.  Since this particular functionality is independent of security and spaces, I figured it would be safe to do there.  There is a test in `security_and_spaces` for create/not-enabled, just not the other HTTP endpoints.

Wouldn’t be a huge hurdle to move ’em all into `security_and_spaces`, but kinda feel like “independent of security” type of functionality, we can get by with the simplest case.  Now that we have a pretty good shape/security base to depend on!

This was perhaps an unusual case because I needed a pre-loaded action SO - an action with a type that’s not enabled, to make sure we can’t execute it (we certainly can’t create it!).  I woulda just done it in `security_and_spaces` but for that.

Reply:
This is good for now, I wrote a note in the cleanup tests suite issue and we can handle the limitation at that time: https://github.com/elastic/kibana/issues/49825.